hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -234,6 +234,16 @@ public final class OzoneConfigKeys {
       OZONE_SNAPSHOT_SST_FILTERING_SERVICE_TIMEOUT_DEFAULT = "300s";
       // 300s for default
 
+  public static final String OZONE_SNAPSHOT_DELETION_SERVICE_INTERVAL =
+      "ozone.snapshot.deletion.service.interval";
+  public static final String
+      OZONE_SNAPSHOT_DELETION_SERVICE_INTERVAL_DEFAULT = "30s";
+
+  public static final String OZONE_SNAPSHOT_DELETION_SERVICE_TIMEOUT =
+      "ozone.snapshot.deletion.service.timeout";
+  public static final String
+      OZONE_SNAPSHOT_DELETION_SERVICE_TIMEOUT_DEFAULT = "300s";
```

Comment:
Just a note that this is not a hard time-out that would stop task execution. AFAIK similar to other tasks this would only [print out a warning](https://github.com/apache/ozone/blob/a66e43e2838f087229cb4b0fac1e1dc969ab80dd/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/BackgroundService.java#L129-L132) if the task run exceed the given time.